geode-core/src/main/java/org/apache/geode/management/internal/cli/i18n/CliStrings.java

```diff
@@ -1800,8 +1803,9 @@
   public static final String PUT__VALUE__HELP =
       "String or JSON text from which to create the value.  Examples include: \"manager\", \"100L\" and \"('value': 'widget')\".";
   public static final String PUT__PUTIFABSENT = "skip-if-exists";
-  public static final String PUT__PUTIFABSENT__HELP =
-      "Skip the put operation when an entry with the same key already exists. The default is to overwrite the entry (false).";
+  public static final String PUT__PUTIFNOTEXISTS = "if-not-exists";
```

Comment:
If you prefer the variable alias `PUT__*`, you could define this instead with 
```
  public static final String PUT__PUTIFNOTEXISTS = IFNOTEXISTS;`
```

The same applies to the `CREATE_REGION_*` one.